package/lang/Default.ini

```diff
@@ -56,7 +56,7 @@ en_mail=sfaulkner@paciellogroup.com
 en_website=https://www.paciellogroup.com/resources/contrastanalyser/
 WATC=http://www.wat-c.org/
 abouttext=The Colour Contrast Analyser was developed by Jun in collaboration with Steve Faulkner.
-versiontext=Colour Contrast Analyser version 2.4
+versiontext=Colour Contrast Analyser version:
```

Comment:
seems that this leaves it dangling as just "Colour Contrast Analyser version:" ?